src/gui/genericerroritemwidget.cpp

```diff
@@ -80,29 +81,33 @@ void GenericErrorItemWidget::init() {
 }
 
 void GenericErrorItemWidget::openFolder(const QString &path) {
-    if (_errorInfo.inconsistencyType() == InconsistencyTypePathLength ||
-        _errorInfo.inconsistencyType() == InconsistencyTypeCase ||
-        _errorInfo.inconsistencyType() == InconsistencyTypeForbiddenChar ||
-        _errorInfo.inconsistencyType() == InconsistencyTypeReservedName ||
-        _errorInfo.inconsistencyType() == InconsistencyTypeNameLength ||
-        _errorInfo.inconsistencyType() == InconsistencyTypeNotYetSupportedChar ||
-        _errorInfo.cancelType() == CancelTypeAlreadyExistLocal ||
-        (_errorInfo.conflictType() == ConflictTypeEditDelete && !_errorInfo.remoteNodeId().isEmpty())) {
+    const auto syncInfoMapIt = _gui->syncInfoMap().find(_errorInfo.syncDbId());
+    if (syncInfoMapIt == _gui->syncInfoMap().end()) {
+        CustomMessageBox msgBox(QMessageBox::Warning, tr("Unable to open folder path %1.").arg(path), QMessageBox::Ok, this);
+        msgBox.exec();
+        return;
+    }
+
+    if (_errorInfo.inconsistencyType() == InconsistencyTypePathLength
```

Comment:
Not important: a helper function with a meaningful name that computes this complex boolean value.

Reply:
Yes, you are right, this is ugly. I am doing it 👍

Reply:
Done